taxonomies/packaging_materials.txt

```diff
@@ -112,211 +203,456 @@ fr:Plastiques mixtes
+description:en:Polyvinylidene chloride is applied as a water-based coating to films made of other plastics, which increases the shelf life of the packaged food.
+description:de:Polyvinylidenchlorid wird als wasserbasierte Beschichtung auf Folien aus anderen Kunststoffen aufgetragen, was die Haltbarkeit der verpackten Lebensmittel erhöht.
+description:es:El cloruro de polivinilideno se aplica como recubrimiento a base de agua a películas hechas de otros plásticos, lo que aumenta la vida útil de los alimentos envasados.
+description:fr:Le chlorure de polyvinylidène est appliqué comme revêtement à base d'eau sur des films faits d'autres plastiques, ce qui augmente la durée de conservation des aliments emballés.
+description:it:Il cloruro di polivinilidene viene applicato come rivestimento a base d'acqua su pellicole di altre materie plastiche, il che aumenta la durata di conservazione degli alimenti confezionati.
+description:nl:Polyvinylideenchloride wordt als coating op waterbasis aangebracht op films van andere kunststoffen, waardoor de houdbaarheid van het verpakte voedsel wordt verlengd.
+description:pt:O cloreto de polivinilideno é aplicado como revestimento à base de água em filmes feitos de outros plásticos, o que aumenta a vida útil dos alimentos embalados.
 
 <en:Polyethylene terephthalate
 <en:Recycled plastic
 en:rPET - Recycled Polyethylene terephthalate, Recycled PET, Recycled Polyethylene terephthalate
-xx:rPET
-de:Wiederverwertetes PET, rPET
+xx:rPET, r-PET
+de:rPET - Wiederverwertetes PET
+es:rPET - Tereftalato de polietileno reciclado
 fr:rPET - Polytéréphtalate d'éthylène recyclé, Polytéréphtalate d'éthylène recyclé, PET recyclé
 hu:rPET - Újrahasznosított PET, Újrahasznosított PET, Újrahasznosított polietilén-tereftalát
-it:PET riciclato, Polietilene tereftalato riciclato
-nl:PET - Hergebruikt
+it:rPET - Polietilene tereftalato riciclato, Polietilene tereftalato riciclato
+nl:rPET - Hergebruikt Polyethyleentereftalaat, Hergebruikt Polyethyleentereftalaat
 pt:rPET - Tereftalato de polietileno reciclado, PET reciclado, Tereftalato de polietileno reciclado
+description:en:rPET (recycled polyethylene terephthalate)
+description:de:Recycelter PET-Kunststoff, hauptsächlich auf Basis von Verbraucherflaschen.
+description:es:Plástico PET reciclado, principalmente a base de botellas de consumo.
+description:fr:Plastique PET recyclé, principalement à base de bouteilles de consommation.
+description:it:Plastica PET riciclata, principalmente basata su bottiglie di consumo.
+description:nl:Gerecycleerde PET plastic, voornamelijk gebaseerd op flessen voor consumenten.
+description:pt:Plástico PET reciclado, principalmente baseado em garrafas de consumo.
 
 <xx:PET
-en:Biobased PET
-de:Biobasiertes PET
-fr:PET biosourcé
-hu:Biobázisú PET
-pt:Bioplástico PET
-xx:Bio-PET
+en:bPET - Biobased PET
```

Comment:
for those, we need to add "Biobased PET" as a synonym (same in other languages)

it's what is causing the tests to fail

Reply:
done